utils/utils.js

```diff
@@ -0,0 +1,11 @@
+const client = require('../config/connection');
+
+exports.internalServerError = (error) => {
+  return { status: 'error', message: error };
+};
```

Comment:
make more specific errors, like server errors, bad requests, unauthorized etc and send specific status codes refer [this](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status) doc for more info